File: app/src/main/java/com/example/birdify/ui/Hotspot/HotspotViewModel.java
package com.example.birdify.ui.Hotspot;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class HotspotViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public HotspotViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is Hotspot fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/example/birdify/ui/Hotspot/HotspotViewModel.kt
package com.example.birdify.ui.Hotspot

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class HotspotViewModel : ViewModel() {
    private val mText: MutableLiveData<String>

    init {
        mText = MutableLiveData()
        mText.value = "This is Hotspot fragment"
    }

    val text: LiveData<String>
        get() = mText
}